src/load.ts

```diff
@@ -82,6 +83,45 @@ export async function load(
     }
 }
 
+/**
+ * Loads the data from a CDN and returns an instance of AzureAppConfiguration.
+ * @param cdnEndpoint  The URL to the CDN.
+ * @param appConfigOptions  Optional parameters.
+ */
+export async function loadFromCdn(cdnEndpoint: URL | string, options?: AzureAppConfigurationOptions): Promise<AzureAppConfiguration>;
+
+export async function loadFromCdn(
+    cdnEndpoint: string | URL,
+    appConfigOptions?: AzureAppConfigurationOptions
+): Promise<AzureAppConfiguration> {
+    const emptyTokenCredential: TokenCredential = {
+        getToken: async () => ({ token: "", expiresOnTimestamp: 0 })
+    };
+
+    // the api version supports sas token authentication
+    const apiVersion = "2024-09-01-preview";
+    const policyName = "CdnRequestApiVersionPolicy";
+    const apiVersionPolicy: PipelinePolicy = {
+        name: policyName,
+        sendRequest: async (request: PipelineRequest, next: SendRequest) => {
+            const url = new URL(request.url);
+            url.searchParams.set("api-version", apiVersion);
+            request.url = url.toString();
+            return next(request);
+        },
+    };
+    if (appConfigOptions === undefined) {
+        appConfigOptions = { clientOptions: {}};
+    }
+    const policies = appConfigOptions.clientOptions?.additionalPolicies || [];
+    // The policy position should be perRetry so that the policy will be processed after the api version policy added by the SDK.
+    // https://learn.microsoft.com/en-us/dotnet/api/azure.core.httppipelineposition?view=azure-dotnet#fields
+    policies.push({policy: apiVersionPolicy, position: "perRetry"});
+    appConfigOptions.clientOptions = { ...appConfigOptions.clientOptions, additionalPolicies: policies};

```

Comment:
Is it bad that this is a new object? If a developer passed in a clientOptions, constructed this, then updated something about the options, it would not reflect here. Not saying that's a common scenario but if the sdk allows for that we probably should too.

Reply:
>  If a developer passed in a clientOptions, constructed this, then updated something about the options, it would not reflect here.

It doesn't sound like a valid use case. The options should be determined when you call the function since you passed the options as a parameter.